docs/insights-archive-sample/config/configmaps/kube-system/cluster-config-v1/install-config.json

```diff
@@ -0,0 +1 @@
+"apiVersion: v1\nbaseDomain: xxxxxxxxxxxxxxxxxxx\ncompute:\n- hyperthreading: Enabled\n  name: worker\n  platform: {}\n  replicas: 3\ncontrolPlane:\n  hyperthreading: Enabled\n  name: master\n  platform: {}\n  replicas: 3\nmetadata:\n  creationTimestamp: null\n  name: dev.rhrmo-dev.7.11.2022\nnetworking:\n  clusterNetwork:\n  - cidr: 10.128.0.0/14\n    hostPrefix: 23\n  machineNetwork:\n  - cidr: 10.0.0.0/16\n  networkType: OVNKubernetes\n  serviceNetwork:\n  - 172.30.0.0/16\nplatform:\n  aws:\n    region: xxxxxxxxx\npublish: External\npullSecret: \"\"\n"
```

Comment:
This doesn't look like valid json file (and from what I've seen locally, it's not a valid one)

Reply:
Will re-naming it to just install-config without the .json at the end be enough? :thinking: